.github/workflows/main.yml

```diff
@@ -78,4 +78,13 @@ jobs:
           DEPLOY_NPM_USERNAME: ${{ secrets.DEPLOY_NPM_USERNAME }}
           DEPLOY_NPM_PASSWORD: ${{ secrets.DEPLOY_NPM_PASSWORD }}
           DEPLOY_NPM_EMAIL: ${{ secrets.DEPLOY_NPM_EMAIL }}
+      - name: Publish chat-plugin library to npm
+        if: ${{ startsWith(github.ref, 'refs/tags/npm-v') }}
```

Comment:
the chat plugin should be in sync with main releases no?

Reply:
yeah, I think it makes more sense